src/components/modules/dropdown/dropdown.styles.ts

```diff
@@ -0,0 +1,64 @@
+'use client';
+
+import Link from 'next/link';
+
+import type { DropdownOptionProps } from './dropdown.types';
+
+import { css, styled } from '@/styles';
+
+const openDropdown = css`
+  padding: 4px;
+  max-height: 300px;
+`;
+
+export const DropdownOptionsWrapper = styled.div`
```

Comment:
Check the design, please. The border is missing and should be the same as the parent button; also, the focus of the parent button is different:

**Figma**
![Screenshot 2024-02-22 at 15 56 23](https://github.com/bejamas/httptoolkit-website/assets/22168189/5771d31c-ee25-471f-af1f-0d398a0d293a)

**Implementation**

![Screenshot 2024-02-22 at 16 00 49](https://github.com/bejamas/httptoolkit-website/assets/22168189/317a8169-d275-4707-ae5f-e1c031417c5b)